### PR TITLE
chore: docs for the github app

### PR DESCRIPTION
**Pull request by ecmadao**
图片

### CODE REVIEW

File: content/docs/en/sql-review/sql-advisor/github-app.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: SQL Advisor GitHub App
+---
+
+Bytebase also provides the [GitHub App](https://github.com/marketplace/bytebase) to let users integrate the SQL review into their GitHub CI.
```
Comment (tianzhou) [inline content/docs/en/sql-review/sql-advisor/github-app.md:5]:
```suggestion
Bytebase also provides the [GitHub App](https://github.com/marketplace/bytebase) to let users integrate the SQL review into their GitHub PR workflow.
```

File: content/docs/en/sql-review/sql-advisor/overview.md [hunk 2]
```diff
@@ -6,4 +6,5 @@ Bytebase provides customizable SQL lint to check common issues in schema change
 
 - [**UI**](/docs/sql-review/sql-advisor/ui) - If you use Bytebase as a whole system, SQL Advisor will check common issues in schema change process;
 - [**API**](/docs/sql-review/sql-advisor/api) - If you have your own database management system, it's possible to integrate SQL Advisor via API;
-- [**GitHub Action**](/docs/sql-review/sql-advisor/github-action) - If all your sql files are stored on GitHub.com, you can use GitHub action to do prechecks before they're merged into the main branch.
\ No newline at end of file
+- [**GitHub Action**](/docs/sql-review/sql-advisor/github-action) - If all your sql files are stored on GitHub.com, you can use GitHub action to do prechecks before they're merged into the main branch.
```
Comment (tianzhou) [inline content/docs/en/sql-review/sql-advisor/overview.md:9]:
```suggestion
- [**GitHub Action**](/docs/sql-review/sql-advisor/github-action) - If all your SQL files are stored on GitHub.com, you can use GitHub action to do prechecks before they're merged into the main branch.
```

File: content/docs/en/sql-review/sql-advisor/overview.md [hunk 3]
```diff
@@ -6,4 +6,5 @@ Bytebase provides customizable SQL lint to check common issues in schema change
...
+- [**GitHub App**](/docs/sql-review/sql-advisor/github-app) - If all your sql files are stored on GitHub.com, you can use GitHub app to do prechecks before they're merged into the main branch. No coding is required, just works in a few seconds of configuration in the UX.
```
Comment (tianzhou) [inline content/docs/en/sql-review/sql-advisor/overview.md:10]:
```suggestion
- [**GitHub App**](/docs/sql-review/sql-advisor/github-app) - If all your SQL files are stored on GitHub.com, you can use GitHub app to do prechecks before they're merged into the main branch. No coding is required, deploy your SQL check in minutes.
```

**Review by tianzhou — APPROVED** [review]
LGTM

**Comment by ecmadao** [comment]
I updated the docs and add the screenshots. PTAL @tianzhou

File: content/docs/en/sql-review/sql-advisor/github-app.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+### Step 1 - Basic info
+
+Specify the basic information, including display name, target repositories, and SQL review policy template.
+
+![app-policy-step1](/static/docs/app-policy-step1.webp)
+
+Repository configuration:
+
+- **Database dialect**: the database dialect for your repository. For now we support MySQL (including TiDB) and PostgreSQL (including CockroachDB)
```
Comment (tianzhou) [inline content/docs/en/sql-review/sql-advisor/github-app.md:33]: Let's use list e.g.

For now, we support:
* MySQL dialect, vanilla MySQL, and other database engines adopting its dialect such as TiDB.
* PostgreSQL dialect, vanilla PostgreSQL, and other database engines adopting its dialect such as CockroachDB.

File: content/docs/en/sql-review/sql-advisor/github-app.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+- **SQL file path**: the file path for the SQL files in the regex format. Only matched files will be checked in the pull request. We
+  will subscribe all SQL files in the repository by default.
+
+![app-config-repo](/static/docs/app-config-repo.webp)
+
+### Step 2 - Configure rules
+
+You can customize the rule level and payload in step 2. You can [follow this doc](/docs/sql-review/review-rules/create-schema-review-policy#step-2) for details about rule configuration.
```
Comment (tianzhou) [inline content/docs/en/sql-review/sql-advisor/github-app.md:41]:
```suggestion
You can customize the rule level and payload in step 2. You can [follow this doc](/docs/sql-review/review-rules/create-schema-review-policy#step-2) for details about the rule configuration.
```

**Review by tianzhou — CHANGES_REQUESTED** [review]

File: content/docs/en/sql-review/sql-advisor/github-app.md (hunk 4 above)
Comment (ecmadao) [reply content/docs/en/sql-review/sql-advisor/github-app.md]: Updated
图片

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**